ksqldb-engine/src/test/java/io/confluent/ksql/function/udaf/offset/EarliestByOffsetTest.java

```diff
@@ -86,7 +89,8 @@ public void shouldThrowExceptionForInvalidN() {
   @Test
   public void shouldComputeEarliestInteger() {
     // Given:
-    final Udaf<Integer, Struct, Integer> udaf = EarliestByOffset.earliestInteger();
+    final Udaf<Integer, Struct, Integer> udaf = EarliestByOffset.earliest();
+    udaf.initializeTypeArguments(Collections.singletonList(SqlArgument.of(SqlTypes.INTEGER)));
```

Comment:
The biggest question I have is if we want to keep these unit tests.  

The previous methods like `earliestInteger` have been removed.  Since `earliest_by_offset` is now configured with the input SQL type, this test is really quite silly looking.  That said, with the modification, it does show that all the various details work.

Should these be left or removed?  Thoughts?

Reply:
I'm +1 to removing it - I don't think it adds much value, especially if we have the QTTs